rust/experimental/server/src/proto/uniffle.proto

```diff
@@ -93,6 +93,7 @@ message GetMemoryShuffleDataRequest {
   int32 readBufferSize = 5;
   int64 timestamp = 6;
   bytes serializedExpectedTaskIdsBitmap = 7;
```

Comment:
`serializedExpectedTaskIdsBitmap` this is the expected bitmap. `BlockIdFilter` is not necessary

Reply:
sure, let me update it.